core/src/sumeragi/mod.rs

```diff
@@ -78,10 +78,10 @@ pub struct Init {
     pub height: u64,
 }
 
-/// Get sorted peers
+/// Get random amount of peers
```

Comment:
```suggestion
/// Get a number of randomly selected peers.
```
"Random amount" means that you have a random usize.